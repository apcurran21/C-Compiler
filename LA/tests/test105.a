void main ( ){

  int64[][] t
  t <- new Array(2,2)
  print(t)
  t[0][1] <- 4
  t[1][1] <- 4

  t[2][0] <- 4

  return 
}
