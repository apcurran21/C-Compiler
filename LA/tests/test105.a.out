{s:6, 2, 2, 0, 0, 0, 0}
ERROR: instruction at line 9 attempted to use position 2 in the dimension 0 of a tensor that only has 2 elements in that dimension
